main/glossary/README.md

```diff
@@ -75,9 +75,12 @@ digital assets in the quantity specified by an [amount](#amounts).
 
 ## BigInt
 
-JavaScript's `Number` primitive only represents numbers up to 253 - 1. `BigInt` is a built-in 
-object that can be used for arbitrarily large integers. Agoric uses `BigInts` for times 
-and `Amount` `values`. See [here](/guides/js-programming/bigint.md) for more information.
+In [ERTP AmountMath](/ertp/guide/amount-math.md), we use the JavaScript [BigInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) type for the `value` of fungible amounts in order to avoid overflow risks from using the usual JavaScript `Number` type.
+
+[Timer Services](/repl/timerServices.md) also use `BigInt` for absolute and relative times.
+
+`BigInt`s are written with an `n` suffix: `0n`, `1n`, `2n`, ... or created with `BigInt("123")`
+or `BigInt(123)`.
```

Comment:
Some caveat approximately like the following seems to be useful:

```suggestion
or `BigInt(123)`.  WARNING: A `Number`, `num` that doesn't satisfy `Number.isSafeInteger(num)` will have rounding errors when you pass it to `BigInt(num)`  When in doubt, use `Nat`.
```

Reply:
We don't seem to document `Nat` at all yet. I'd like to postpone this as #586.